src/v/cluster/controller_backend.cc

```diff
@@ -140,6 +140,8 @@ controller_backend::controller_backend(
   , _topics_frontend(frontend)
   , _self(model::node_id(config::shard_local_cfg().node_id))
   , _data_directory(config::shard_local_cfg().data_directory().as_sstring())
+  , _housekeeping_timer_interval(
+      config::shard_local_cfg().backend_housekeeping_interval_ms())
```

Comment:
this is fine for this PR. but we are trying to move away from depending on this global state as it makes things hard to test. 

can you create a tix and self assign to remove his dependency

Reply:
meaning add a struct. for options and do this at constructor

Reply:
it will have to change a little bit. creating the copy of properties prevents us from using runtime changed configuration.

Reply:
> ... prevents us from using runtime changed configuration.

the controller backend is instantiated once, so i don't see the argument for this

Reply:
When property would change in the configuration, we have to somehow update it in backend, we can either notify backend about changes or make it poll the configuration directly.